docs/Testing.md

```diff
@@ -29,17 +33,15 @@ get everything working.
 
 ### Run local DynamoDB
 
-We need to set the `TESTING` to indicate which DynamoDB (local or server) is
-ran. `TESTING` is default to false. When `TESTING=false`, we indicate we want to
-run the server DynamoDB.
+We use `testing` in the `config.toml` to indicate if we want to run DynamoDB
+locally or on a server. Change `testing = true` to use local DynamoDB.
 
-`TESTING=false pytest`
+This is the recommended way for unit testing.
 
 ### Run server DynamoDB
 
-To run the server DynamoDB we need to set the `REGION`. The `REGION` is default
-to `us-east-1`, however you can change it to [any of the regions
-here](https://docs.aws.amazon.com/general/latest/gr/rande.html).
+To run the server DynamoDB we need to set the `region` and obtain both the
+AWS `access_key_id` and `secret_access_key`.
```

Comment:
May want to include a pointer to the full local development setup documentation that explains this.